deepchem/feat/graph_data.py

```diff
@@ -90,7 +90,26 @@ def __init__(
     self.num_edges = edge_index.shape[1]
     if self.edge_features is not None:
       self.num_edge_features = self.edge_features.shape[1]
+  
+  def __repr__(self):
+    """Utility to print the graph in more useful way.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> node_features = np.random.rand(5, 10)
+    >>> edge_index = np.array([[0, 1, 2, 3, 4], [1, 2, 3, 4, 0]], dtype=np.int64)
+    >>> graph = GraphData(node_features=node_features, edge_index=edge_index)
+    >>> print(graph)
 
+    Returns
+    -------
+        GraphData(node_features=[5, 10], edge_index=[2, 5])
+     """
+    node_features_str = str(list(node_features.shape))
+    edge_index_str = str(list(edge_index.shape))
+    return  "GraphData(node_features="+node_features_str+", edge_index="+edge_index_str+")"
```

Comment:
You need to use string formatting here before returning here. It improves readability.